crates/fluvio-version/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+/// The `git rev-parse HEAD` of the current build of Fluvio.
+pub const GIT_HASH: &str = env!("GIT_HASH");
+
+/// The current version of Fluvio.
+pub const VERSION: &str = include_str!("../../../VERSION");
```

Comment:
This could be a `semver::Version`, enough dependents use it like that.

Reply:
Also it's notable here, `include_str` adds a "build if changed" style condition on included files.

Reply:
This would be nice!

It also would be nice to have a ENV VAR to solve workspace dir so we don't have to write 3 times the `../` bit, but AFAIK its not available.

Reply:
I may have misremembered wrt `semver::Version`, it seems like only `fluvio-test-util` parses it and some of the others sort of need it anyway as a literal string.

I agree about the env var, although this line should refuse to compile if the file moves, and now we only have one reference in the rust code to this path instead of 15 or so.